package.json

```diff
@@ -107,6 +110,16 @@
     "test:prepare": "yarn clear-out && tsc && yarn test:copyassets",
     "test": "yarn test:prepare && node ./out/test/runTest.js",
     "vsce:package": "vsce package",
+    "onprem:prep": "yarn onprem:prep:package && yarn onprem:prep:readme",
```

Comment:
would suggest to do all of it in a `buildOnPrem.js` file because it is not readable, this file shold be not coompiled

Reply:
because there are shell commands here and this mechanism allows for running from outside individually